.travis.yml

```diff
@@ -26,4 +26,4 @@ env:
   - CAMUNDA=7.11.0
   - CAMUNDA=7.10.0
 install: true
-script: mvn -Dcamunda.version=$CAMUNDA -Pinclude-examples
+script: mvn -Dcamunda.version=$CAMUNDA -Pinclude-examples com.mycila:license-maven-plugin:check
```

Comment:
Unfortunately, this leads to ONLY the license check being executed, since the project works with default build goals `clean install`. I will adjust the YAML to include those two goals explicitly as well.